src/main/java/io/jenkins/plugins/appcenter/api/AppCenterServiceFactory.java

```diff
@@ -87,4 +109,26 @@ public UploadService createUploadService() {
     public Secret getApiToken() {
         return apiToken;
     }
+
+    private void setProxy(
+        final ProxyConfiguration cfg,
```

Comment:
Really small point here, but do you mind changing this variable name to something more meaningful please. `configuration` / `proxyConfiguration` would do.

Reply:
Changed.